public/app/plugins/panel/alertlist/module.ts

```diff
@@ -125,7 +125,7 @@ class AlertListPanel extends PanelCtrl {
       .then(res => {
         this.currentAlerts = this.sortResult(_.map(res, al => {
           al.stateModel = alertDef.getStateDisplayModel(al.state);
-          al.newStateDateAgo = moment(al.newStateDate).fromNow().replace(" ago", "");
+          al.newStateDateAgo = moment(al.newStateDate).locale('en').fromNow(true);
```

Comment:
because "fromNow(true)" returns "4 hours" and you don't need to do string replacement.
See https://momentjs.com/docs/#/displaying/fromnow/